rules/@typescript-no-watch/@typescript-no-watch.md

```diff
@@ -0,0 +1,108 @@
+# effector/@typescript-no-watch
+
+Method `.watch` leads to imperative code. Try replacing it with operators (`forward`, `sample`, etc) or use the `target` parameter of the operators.
+
+```ts
```

Comment:
Such number of examples leads to poor documentation reading experience. Could we replace it with a few amount of the most impressive cases?